styles/all/template/event/forumlist_body_category_header_before.html

```diff
@@ -0,0 +1,7 @@
+<!-- IF forumrow.S_IS_CAT -->
+<a class="category <!-- IF forumrow.S_FORUM_HIDDEN -->hidden-category<!-- ENDIF -->"
```

Comment:
We should move the space after `category` to inside the conditional before `hidden` so that it renders as `class="category"` when there is no hidden category